cmd/immuclient/cli/references_test.go

```diff
@@ -59,27 +35,11 @@ func TestReference(t *testing.T) {
 	}
 }
 
-func _TestSafeReference(t *testing.T) {
-	options := server.DefaultOptions().WithAuth(true)
-	bs := servertest.NewBufconnServer(options)
-
-	bs.Start()
-	defer bs.Stop()
-
-	defer os.RemoveAll(options.Dir)
-	defer os.Remove(".state-")
+func TestSafeReference(t *testing.T) {
+	t.SkipNow()
```

Comment:
I'm afraid I don't know the reason why this test was disabled in the first place. The only think I did in this PR was to conver the `_Test...` to a normal test with `t.Skip()` so that it is not completely ignored (and there's a ton of other tests out there that have this which I didn't convert). There's a bit of mess in those tests too, hopefully we'll slowly get there.

I also tried to figure out the reason for disabling this test through git history and this is what I've found: https://github.com/codenotary/immudb/commit/beb9fbba1e422bd9cff36226cbab7add932a6ce7

`fix(cmd/immuclient): fix unit tests` :rofl: